app/views/case_contacts/_case_contact.html.erb

```diff
@@ -35,27 +35,29 @@
           </div>
         </div>
       </div>
-      <div class="d-flex justify-content-between">
+      <div class="d-flex justify-content-between align-items-center">
         <% if Pundit.policy(current_user, contact).update? %>
           <%= render "case_contacts/followup", contact: contact, followup: contact.requested_followup %>
           <div class="mr-2">
-            <%= link_to edit_case_contact_path(contact), class: "btn btn-outline-primary" do %>
-              <strong>Edit</strong>
+            <%= link_to edit_case_contact_path(contact), class: "text-danger h5" do %>
+              <i class="lni lni-pencil-alt"></i> Edit
             <% end %>
           </div>
         <% end %>
-        <div class="">
-        <div class="">
-          <%= link_to("Delete", case_contact_path(contact.id), method: :delete,
-                class: "btn btn-danger") if policy(contact).destroy? && !contact.deleted? %>
+        <div class="action">
+        <!----- Couldn't find a way to include this conditional block
+                if policy(contact).destroy? && !contact.deleted? 
+                in the link_to below                            ----->
+         <%= link_to case_contact_path(contact.id), class: "main-btn btn-sm danger-btn btn-hover", method: :delete  do %>
```

Comment:
You couldn't put the whole `link_to` inside the if block?
like on line 50 you:
`<% if policy(contact).destroy? && !contact.deleted? %>`
and on line 54:
`<% end %>`
Doesn't work?

Reply:
Oh I see, I can definietly wrap the link_to block with the conditional. I just wasn't 100% sure because I hadn't seen a conditional within a link_to block the way it was originally implemented and didn't know if extracting it would change the behavior. Thank you!